source/doi_howto.rst

```diff
@@ -0,0 +1,124 @@
+How-to
+""""""
+
+Preparing a Dataset
+===================
+Below, the steps required to prepare and expose a dataset are briefly overviewed, together with the steps necessary to have a DOI assigned by METNO's DOI registration agency, i.e. DataCite. 
```

Comment:
MET Norway's

Reply:
Thanks! I'll replace this, also in other parts of the commits.